Packs/MicrosoftGraphCalendar/Integrations/MicrosoftGraphCalendar/MicrosoftGraphCalendar.py

```diff
@@ -699,7 +577,8 @@ def main():
     LOG(f'Command being called is {command}')
 
     try:
-        client = Client(url, tenant, auth_and_token_url, auth_id, token_retrieval, enc_key, verify, proxy, default_user)
+        client: MsGraphClient = MsGraphClient(tenant, auth_and_token_url, enc_key, APP_NAME,
```

Comment:
map each argument you are passing, so it will be more readable
for example `(tenant_id=tenant, auth_id= auth_and_token_url, ...)`